src/assets/foot_restraints/leather_ankle_cuffs/leather_ankle_cuffs.asset.ts

```diff
@@ -110,6 +110,48 @@ DefineAsset({
 						},
 					},
 				},
+				{
+					id: 'short',
+					name: 'Short Chain',
+					properties: {
+						poseLimits: {
+							bones: {
+								leg_r: [[0, 2]],
+								leg_l: [[0, 2]],
+							},
+						},
+						stateFlags: {
+							provides: ['chain'],
+						},
+					},
+				},
+				{
+					id: 'clipped',
+					name: 'Clipped together',
+					properties: {
+						poseLimits: {
+							options: [
+								{
+									legs: 'standing',
+									bones: {
+										leg_r: 2,
```

Comment:
2/2 does not only work for standing, but also for sitting in my tests.

I think there is no reason to not forbid kneeling in the clipped together state.

Reply:
The clipped state in kneeling requires bones of 6/6 (lines 142ff in the same file)